en/resources/tutorials/codablerouting.md

```diff
@@ -0,0 +1,172 @@
+---
+### TRANSLATION INSTRUCTIONS FOR THIS SECTION:
+### TRANSLATE THE VALUE OF THE title ATTRIBUTE AND UPDATE THE VALUE OF THE lang ATTRIBUTE.
+### DO NOT CHANGE ANY OTHER TEXT.
+layout: page
+title: Codable Routing
+menu: resources
+lang: en
+redirect_from: "/resources/codablerouting.html"
+### END HEADER BLOCK - BEGIN GENERAL TRANSLATION
+---
+
+<div class="titleBlock">
+  <h1>Codable Routing</h1>
+  <p>How to use Codable Routing in Kitura</p>
+</div>
+
+
+
+## Introduction 
+Kitura 2.0 introduces “Codable Routing”. Here the the router handlers are much like normal functions you might define elsewhere in your code: they take struct or class types as parameters, and respond with stuct or class types via a completion handler, with just an additional requirement that those types implement [Codable](https://developer.apple.com/documentation/swift/codable) from Swift 4 (hence the name).
+
+Codable Routing isn’t suitable for every use case and scenario, so Raw Routing is still available where you need the power and flexibility, but it is perfect for building REST APIs such as you might want to do to build a Backend For Frontend (BFF) for an iOS app.
+
+## Building a Create API with a Codable Route for POST requests
+
+Building a Codable Route for a POST API to implement storing a To Do item requires only 4 lines of code. The first step is to register a handler for POST requests on “/todos”:
```

Comment:
The '4 lines of code' mentioned here isn't immediately referenced or shown straight after, which tends to make the user try count the number of lines of code excerpts further down. Consider showing the 4 lines of code immediately after, before the code for registering a handler, or mention this later immediately before the bit with a distinct 4 lines.